index.bs

```diff
@@ -148,6 +148,23 @@ this header if it contains an invalid value. Likewise, user agents MUST ignore t
 value cannot be parsed as a <a grammar>`sh-token`</a>.
 
 
+The `Cross-Origin-Embedder-Policy-Report-To` HTTP Response Header {#COEP-REPORT-TO}
+----------------------------------------------------------------
+
+The <dfn http-header>`Cross-Origin-Embedder-Policy-Report-To`</dfn> HTTP response header field
+allows a server to specify an endpoint for reporting potential policy violations. It is a
+[=Structured Header=] whose value MUST be a [=structured header/token=].
+[[!I-D.ietf-httpbis-header-structure]] Its ABNF is:
+
+```
+Cross-Origin-Embedder-Policy-Report-To = sh-token
```

Comment:
Why would we do this as a separate header rather than a parameter on the existing header? That is, `Cross-Origin-Embedder-Policy: require-corp; report-to=group` is simple, consistent with CSP, and (IMO) aesthetically better than creating a new header.

@annevk, as I think you prefer the way it's defined here. :)

Reply:
Done.